app/assets/engine.js

```
@@ -137,7 +149,16 @@ export function processTurn() {
 	const activeEntities = [...entities]; // Create a shallow copy to iterate over
 
 	for (const e of activeEntities) {
-		if (!e.mob || e.mob.ai === "dead") continue;
+		if (e.mob && e.mob.ai === "dead") {
+			// --- NEW: INCREMENT KILL COUNT ---
+			if (e.owner !== "player") {
+				// Only count non-player deaths
+				VARS.killCount++;
+			}
+			// --- END ---
+			dead_entities_this_turn.push(e);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)